### ISSUE TITLE
Incomplete documentation information

### ISSUE THREAD

**Issue by asvinours** [regel/loudml#37]
Hi,

I have two main questions, tried to search everywhere in the documentation and I can't find the answers:
`offset | (duration) The time offset used when querying the data source`
I am having a really hard time understand what this mean. Could you provide more information?

`One or more match_all conditions can be added and will automatically change the queries to your data sources with the right filters.`
Is this feature only available for InfluxDB datasource? I tried to get it to work for elasticsearch and the filter doesn 't seem to work properly. When using the predict API, the observed count doesn't match the expected value if the filter was working properly.
```
{
  "bucket_interval": "5m",
  "default_datasource": "elastic",
  "timestamp_field": "createddate",
  "features": [
    {
      "default": 0,
      "metric": "count",
      "field": "attempted",
      "measurement": "item",
      "name": "count_all_submitted_jobs",
      "anomaly_type": "low_high",
      "match_all": [
        {"tag": "extra.submit_type", "value": "submit"}
      ]
    }
  ],
  "seasonality": {
      "daytime": true,
      "weekday": true
  },
  "interval": 300,
  "max_evals": 10,
  "name": "submitted-lh-model",
  "offset": 120,
  "forecast": 30,
  "span": 30,
  "max_threshold": 25,
  "min_threshold": 10,
  "type": "timeseries"
}
```

I used strace while running the predict API and I cansee there is no filter in the elasticsearch query:
```
{\"query\":{\"bool\":{\"must\":[{\"range\":{\"createddate\":{\"lt\":1534382700000,\"gte\":1534290300000}}}]}},\"aggs\":{\"histogram\":{\"date_histogram\":{\"extended_bounds\":{\"min\":1534290300000,\"max\":1534382400000.0},\"min_doc_count\":0,\"field\":\"createddate\",\"interval\":\"300s\",\"order\":{\"_key\":\"asc\"}},\"aggs\":{\"count_all_submitted_jobs\":{\"stats\":{\"field\":\"attempted\"}}}}}
```

**Comment by asvinours** [comment]
Hi,

I am using the DEB package:
loudml:
  Installed: 1.3.2
  Candidate: 1.3.2
  Version table:
 *** 1.3.2 500
        500 http://loudml.s3-website-eu-west-1.amazonaws.com/repo/deb stretch/main amd64 Packages

Thank you for the information about the offset, I understand a lot better now.

**Comment by regel** [comment]
You're welcome.
Thank you for release information. Actually version 1.4.0 will be released in a couple days with several enhancements. We will try to include the fix in this version and keep you updated.

**Comment by regel** [comment]
Fixed in 1.4.0, release notes: http://loudml.io/guide/en/loudml/reference/current/release-notes-1.4.0.html

Fabrice is it ok in your environment after the upgrade to 1.4.0?

**Comment by regel** [comment]
@asvinours : documentation updated, and a new release (1.4.2) is now available. We've made small changes to the ES API, in order to support the doc_type in config.yml - since new ES only allows 1 doc_type per index. It may impact your current settings. The 1.4.2 release notes is available, let us know your comment.
